payment_cycle/migrations/0002_add_pc_rights_to_admin.py

```diff
@@ -0,0 +1,34 @@
+from django.db import migrations
+
+benefit_plan_rights = [170001, 170002, 170003, 170004]
```

Comment:
Should be 200001 and 200005 only? @malinowskikam

Reply:
Since they are already in migration we can stick to 170000. I updated https://openimis.atlassian.net/wiki/spaces/OP/pages/1775173658/openIMIS+Authorities and front perms.

Reply:
No, please do not use 170000 because it is already used in Beneficiares @jdolkowski

Reply:
You are right. Let's stick to 200000. I didn't use space when ctrl+f

Reply:
I fixed the migration and added the rest of the rights (200001-20005) for documentation purposes.